html-web-page/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>Here at JOTMC Inc., we are constantly striving to supply our customer with the highest-quality
+    and most unique jellies on a monthly basis. For the jelly lover in your life (or yourself!),
+    this is the gift that keeps on giving all year around. We offer three tiers of monthly jello joy, and each one
+    offers their own perks. Here is the breakdown of each, as well as your monthly price: </p>
+
+  <table>
+    <thead>
+      <tr>
+        <td></td>
+        <td>Basic Plan</td>
+        <td>Duluxe Plan</td>
+        <td>Jelly Giant Plan</td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new curated jellies per month</td>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commemorative mason jars</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free bounus jar of preserver bi-monthly</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to Jellt Con</td>
+        <td></td>
+        <td></td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <form action="">
+   <div>
+     <div>
+    <label for="sign-up">Like what you see? Become part of the JOTMC family today!</label>
+   </div>
+    </div>
+   <div>
+  <label>
```

Comment:
This `label` element is not needed, since there is no label for this form control element 😁